cpp/src/request_tag.cpp

```diff
@@ -107,15 +113,15 @@ void RequestTag::request()
     request       = ucp_tag_send_nbx(_endpoint->getHandle(),
                                _delayedSubmission->_buffer,
                                _delayedSubmission->_length,
-                               _delayedSubmission->_tag,
+                               *_delayedSubmission->_data._tag,
```

Comment:
**question:** Is anyone checking that `_data._tag.has_value()`? If not then this is potentially UB if we derefence `std::nullopt`.

Reply:
Not relevant anymore after https://github.com/rapidsai/ucxx/pull/121/commits/5037e8fd31a0edce8be8f204619955566c3001ef.